go.sum

```diff
@@ -749,11 +749,13 @@ github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANyt
 github.com/influxdata/influxdb1-client v0.0.0-20191209144304-8bf82d3c094d/go.mod h1:qj24IKcXYK6Iy9ceXlo3Tc+vtHo9lIhSX5JddghvEPo=
 github.com/ishidawataru/sctp v0.0.0-20190723014705-7c296d48a2b5/go.mod h1:DM4VvS+hD/kDi1U1QsX2fnZowwBhqD0Dk3bRPKF/Oc8=
 github.com/itchyny/go-flags v1.5.0/go.mod h1:lenkYuCobuxLBAd/HGFE4LRoW8D3B6iXRQfWYJ+MNbA=
+github.com/itchyny/go-flags v1.5.0/go.mod h1:lenkYuCobuxLBAd/HGFE4LRoW8D3B6iXRQfWYJ+MNbA=
```

Comment:
This (and below) are exact duplicates. is this exepcted?

Reply:
We already have one example on the master branch that was not introduced by Dependabot https://github.com/DataDog/datadog-agent/blob/dd41dd75294f4605409aea7545c26e61f77d7abb/go.sum#L755-L756 Our `go mod tidy` check passes, so I would say it's fine

Reply:
I think those were not added on this PR but Github shows in the diff for some reason, I asked Dependabot to recreate the PR and they should go away

Reply:
Fixed